fabfile.py

```diff
@@ -176,7 +178,8 @@ def ec2task():
 
 def ec2():
     env.user = 'ubuntu'
-    env.key_filename = ['/Users/sclay/.ec2/sclay.pem']
+    env.key_filename = '/Users/%(user)s/.ec2/$(user).pem' % {
+        'user': getpass.getuser()}
```

Comment:
Since env.user is being overwritten with ubuntu, you'll have to use a variable that you set above.